SwiftBar/AppDelegate.swift

```diff
@@ -125,17 +125,23 @@ class AppDelegate: NSObject, NSApplicationDelegate, SPUStandardUserDriverDelegat
                                                title: url.queryParameters?["title"],
                                                subtitle: url.queryParameters?["subtitle"],
                                                body: url.queryParameters?["body"],
+                                               open: url.queryParameters?["open"],
                                                silent: url.queryParameters?["silent"] == "true")
             default:
                 os_log("Unsupported URL scheme \n %{public}@", log: Log.plugin, type: .error, url.absoluteString)
             }
         }
     }
 
-    func userNotificationCenter(_: UNUserNotificationCenter, willPresent _: UNNotification,
-                                withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void)
-    {
-        completionHandler([.alert, .sound])
+    func userNotificationCenter(_: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+        let urlString = response.notification.request.content.userInfo["url"] as? String ?? ""
+
+        if urlString != "" {
+            let url = URL(string: urlString)!
+            NSWorkspace.shared.open(url)
+        }
+
```

Comment:
I try to avoid explicit unwrapping whenever it is possible, a cleaner version of this will look like 

```swift
if let urlString = response.notification.request.content.userInfo["url"] as? String,
   let url = URL(string: urlString) {
   NSWorkspace.shared.open(url)
}
```

Reply:
Changed